schedule/yast/lvm/lvm_sle_x86_64.yaml

```diff
@@ -0,0 +1,32 @@
+---
+description: >
+  Conduct installation with LVM selected during installation using guided setup.
+  In comparison to SLE 12 we register the installation and have system roles
+  wizard screen.
+name: cryptlvm
+vars:
+  DESKTOP: textmode
+  LVM: '1'
+  MAX_JOB_TIME: '14400'
+  OPT_KERNEL_PARAMS: console=%SERIALDEV% VNCSize=1024x768
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/skip_module_registration
+  add_on_product:
+    - installation/add_on_product/skip_install_addons
```

Comment:
In the adaption in [PR#16218](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16218) this will be the default already.